recipes/bzip2dynamic/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0.6" %}
+
+
+package:
+  name: bzip2
+  version: {{ version }}
```

Comment:
the name should always reflect the foldername, but this package already exists in conda-forge. So no need to include it here.